src/components/Form/FormFields/Autocomplete.tsx

```diff
@@ -235,25 +242,34 @@ export const Autocomplete = <T, V>(props: AutocompleteProps<T, V>) => {
                     disabled={option.disabled}
                   >
                     {({ focus }) => (
-                      <div className="flex flex-col">
-                        <div className="flex justify-between">
-                          <span>{option.label}</span>
-                          <span>{option.icon}</span>
-                        </div>
-                        {option.description && (
-                          <div
-                            className={classNames(
-                              "text-sm font-normal",
-                              option.disabled
-                                ? "text-secondary-700"
-                                : focus
-                                  ? "text-primary-200"
-                                  : "text-secondary-700",
-                            )}
-                          >
-                            {option.description}
-                          </div>
+                      <div className="flex flex-row gap-2">
+                        {option?.image && (
+                          <Avatar
+                            className="h-11 w-11 rounded-full"
+                            name={option.label}
+                            imageUrl={option.image}
+                          />
```

Comment:
Let's say a user doesn't have an image but another user does. For the one's that doesn't, we should show the initials right? Otherwise this would cause inconsistent alignements right?

Reply:
Yea, doesn't look too off, but you are right, better to display the initials 🤔  

<img width="846" alt="image" src="https://github.com/user-attachments/assets/1176eecb-400c-479e-a58c-e4196db9a5dc">

Reply:
yup, we could simply refactor `optionImage` to return a `ReactNode` instead of a `string` (image URL) and in UserAutocomplete return the Avatar.